build/openrpc.go

```diff
@@ -0,0 +1,30 @@
+package build
+
+import (
+	"encoding/json"
+
+	rice "github.com/GeertJohan/go.rice"
+)
+
+type OpenRPCDocument map[string]interface{}
+
+func OpenRPCDiscoverJSON_Full() OpenRPCDocument {
+	data := rice.MustFindBox("openrpc").MustBytes("full.json")
+	m := OpenRPCDocument{}
+	_ = json.Unmarshal(data, &m)
+	return m
+}
+
+func OpenRPCDiscoverJSON_Miner() OpenRPCDocument {
+	data := rice.MustFindBox("openrpc").MustBytes("miner.json")
+	m := OpenRPCDocument{}
+	_ = json.Unmarshal(data, &m)
+	return m
+}
+
+func OpenRPCDiscoverJSON_Worker() OpenRPCDocument {
+	data := rice.MustFindBox("openrpc").MustBytes("worker.json")
```

Comment:
Given that those are large-ish (1MiB), I'd gzip those files before embedding into the binary (not that 1MiB is a lot with our 100M+ binaries, but still worth trying to save some space)

Reply:
This was addressed with https://github.com/filecoin-project/lotus/pull/4711/commits/145e8bb4125c72fc6cbdbf604e8e404fa38c44aa and the couple subsequent commits.